app/views/indexes/show.haml

```diff
@@ -25,7 +25,7 @@
       %tr
         %td.odd.written-content{colspan: 6}= sanitize_written_content(@index.description).html_safe
   %tbody
-    = render partial: 'index_sections/single', collection:@index.index_sections.ordered, as: :section
+    = render partial: 'index_sections/single', collection: @index.index_sections.ordered, as: :section
```

Comment:
Anything using collection or as arguments can't have the partial keyword removed or it assumes the arguments are locals not arguments for render. There's plenty of places I left alone, this one just showed up cause I added the missing space, so figured I'd call out the case.